Bixal/move.mil
# add scss linter with custom config

This PR contains support for command-line sass linting through stylelint. It's run as part of the watch/build process from the theme, set up with npm scripts. It also includes custom lint config that I put together after exploring a few different options.

Also, settings.php is updated to allow local theme overrides - which is necessary to avoid clearing cache for every css (and eventually js) change.

To test:

1. Pull this branch (obv)
1. go to the theme folder, '<project root>/web/themes/custom/move_mil'
1. run 'npm install'
1. run 'npm run watch'
1. keep watch the terminal for linting results. It should return a few warnings (that will be fixed in a future PR within this sprint).
1. Make a modest change within the '_variables.scss' file and save it. The lint and build should both automatically rerun.
